src/OmniSharp.Abstractions/Models/v2/SemanticHighlight/SemanticHighlightModifier.cs

```diff
@@ -0,0 +1,6 @@
+namespace OmniSharp.Models.SemanticHighlight
+{
+    public enum SemanticHighlightModifier
```

Comment:
this doesn't seem to be used?

Reply:
Although the work isn't planned, we can expect that Rolsyn will introduce a classification API which provides this additional information about classified spans. I wanted the API to reflect it so that consumers could expect it being returned. I could probably process the classifications and return a static modifier...